### ISSUE TITLE
In the Bard (Web) model, questions remain unanswered with a flashing message stating "Waiting response..." indefinitely.

### ISSUE THREAD

**Issue by tonylost** [josStorer/chatGPTBox#639]
Bug Description:
In the Bard (Web) model within the extension, whenever I click on the extension icon and pose any question in the floating panel, all responses remains indefinitely with a flashing message stating "Waiting response...".

Steps to Reproduce:

Install the extension on your browser.
Navigate to a webpage where the extension is applicable.
Click on the extension toolbar icon to open the floating panel.
Pose any question within the floating panel.
Observe the indefinite flashing message "Waiting response..." in the Bard (Web) model.
Expected Behavior:
Responses to questions posed within the floating panel should be generated promptly without indefinite waiting periods.

Actual Behavior:
In the Bard (Web) model, questions remain unanswered with a flashing message stating "Waiting response..." indefinitely.

Additional Information:

Browser: Google Chrome
Extension Version: 2.4.9
Operating System: Windows

Error Screenshots:
![2024-03-03 19_23_11-New Issue · josStorer_chatGPTBox](https://github.com/josStorer/chatGPTBox/assets/835437/8c02b516-1df5-4cc2-9e50-94a1cf1b5735)

Please let me know if you require any further information or if there are specific steps you'd like me to take to assist in resolving this issue. I appreciate your prompt attention to this matter.

**Comment by duAffentier** [comment]
https://github.com/josStorer/chatGPTBox/issues/627

This?

**Comment by josStorer** [comment]
Due to the gradual cessation of maintenance for many open-source projects in reverse engineering, I am alone responsible for reverse engineering multiple websites' API, requiring a considerable amount of effort. Therefore, currently, I prioritize maintaining reverse engineering APIs for ChatGPT, Claude, and Bing

**Comment by tonylost** [comment]
> Due to the gradual cessation of maintenance for many open-source projects in reverse engineering, I am alone responsible for reverse engineering multiple websites' API, requiring a considerable amount of effort. Therefore, currently, I prioritize maintaining reverse engineering APIs for ChatGPT, Claude, and Bing

I get the prioritization you're talking about. Now Bard has also been updated to Gemini. Maybe that's the issue and a simple update could fix it.

**Comment by tonylost** [comment]
> #627
> 
> This?

Yes, thank you, in this case the error is exactly what's happening. I went ahead and tried the new update 2.5.0 of the extension and it's fixed. It will soon update automatically in the Chrome Store too.
https://github.com/josStorer/chatGPTBox/commit/1fc3cd8ee072d420fc54e4ce01a6853c71c82927

**Comment by wwjCMP** [comment]
In fact, is supporting Gemini pro a better choice?  Because it provides free api

**Comment by josStorer** [comment]
[v2.5.1](https://github.com/josStorer/chatGPTBox/releases/tag/v2.5.1)

**Comment by duAffentier** [comment]
Hi its dosent work...same error...
ne respon from gpt
